src/api/include/migraphx/migraphx.hpp

```diff
@@ -777,6 +777,13 @@ struct context
     migraphx_context_t ctx;
 
     void finish() const { call(&migraphx_context_finish, ctx); }
+
+    void* get_queue()
```

Comment:
This should do the cast so we can type check this in the future:

```cpp
    template<class T>
    T get_queue() const
    {
        void* out;
        call(&migraphx_context_get_queue, &out, ctx);
        // TODO: Check type
        return reinterpret_cast<T>(out);
    }
```

Reply:
made changes.

Reply:
It needs to return `T` not `void*`.

Reply:
Done.